app/src/test/java/org/simple/clinic/recentpatient/RecentPatientsScreenControllerTest.kt

```diff
@@ -110,7 +110,7 @@ class RecentPatientsScreenControllerTest {
     uiEvents.onNext(ScreenCreated())
 
     verify(screen).updateRecentPatients(listOf(
-        DateHeader(updatedAt1.toLocalDateAtZone(utcClock.zone)),
+        DateHeader(org.simple.clinic.recentpatient.Today),
```

Comment:
Can you update this test to have patients segregated according to days? Right now, only `Today` is being tested.

Reply:
Ah, what I meant was that we should test that the segregation of patients into groups according to their updated at date must be verified. Let's pair on this later.

Reply:
Sure